client/src/components/Allocations/AllocationsModals/AllocationsManageTeamTable/AllocationsManageTeamTable.module.scss

```diff
@@ -6,36 +6,57 @@
   padding-right: 0;
 }
 .manage-team-table {
-  font-size: 0.875rem;
-  line-height: 2rem;
-  margin: 0 1rem;
-  thead {
-    border-bottom: 1px solid black;
-  }
+  font-size: 12px; /* 10px design * 1.2 design-to-app ratio */
+  margin-bottom: 0; /* overwrite bootstrap */
+  border-collapse: separate; /* keep <th> border-bottom visible on scroll */
+  border-spacing: 0;
```

Comment:
- The `font-size` at `0.875rem` is 14px (too big)
- The `line-height` made wrapped cell text have too much space between lines.
    - Instead, I updated cell padding (a few lines after this block).
- The `margin` was replaced with default modal margin (made available by removing global `p2` class).
- The `thead` border is moved to `th`.
    - Because otherwise border does not work with `o-fixed-header-table`.

Reply:
> The font-size at 0.875rem is 14px (too big)

Why's that? I thought it looked ok myself

Reply:
The margin bottom may have been to add space to bottom of modal. I'll add it another way in a new commit.

Reply:
Fixed validation (7fcfddd) and spacing (c82db8c).

Reply:
Font-size is "too big" compared to font-size in design (multiplied by 1.2 font-size multiplier[^1]). It may have looked good, because it matched the View Team modal.

<details>

![Font Size in Design (Needs 1 2 Multiplier)](https://user-images.githubusercontent.com/62723358/173453062-e5cff252-5c56-4141-9a94-afa4dc785ef3.png)

</details>

This all said… I remember a major Design-lead/Dev-approved font-size increase months ago, recently. Uh oh, maybe it should be 14px.[^2]

[^1]: Confluence down, so I can't show document for this, but ` * 1.2` is mentioned in (too) many comments in the code base.
[^2]: Wes, does this mean the multiplier should have changed?

Reply:
Ah, table font sizes _were_ increased across the board (https://github.com/TACC/Core-Portal/pull/401). Well, new To Do item.

Reply:
Font size reverted (9f21ad5).

Reply:
@wesleyboar Does the font size reversion warrant an adjustment to padding?

Reply:
I don't think so, because no padding was adjusted in the font size increase at [#401](https://github.com/TACC/Core-Portal/pull/401/files). **But** the vertical padding I added was _too small_ even with 12px font size. Good eye. I've fixed the padding (7c99daf) and updated description screenshots.

<details><summary>Screenshots of Dimensions</summary>

| Design Padding | 12px Font, Wrong Padding | 14px Font, Wrong Padding |
| - | - | - |
| ![Design Dimensions](https://user-images.githubusercontent.com/62723358/173602675-d06f21c1-2cb4-47ef-b3af-9871c0827f5b.png) | ![Font Size 12px](https://user-images.githubusercontent.com/62723358/173602475-5824e50a-87c8-43f1-b8dd-a5da73203fc3.png) | ![Font Size 14px](https://user-images.githubusercontent.com/62723358/173602485-10dbf9ea-7c28-4ac9-8c9f-4f4a5e20c132.png) |

</details>